auction-app-frontend/src/constants.js

```diff
@@ -41,3 +41,6 @@ export const HIDE_BREADCRUMBS_ON_PATHS = [
   ROUTE_PATHS.HOME,
   // will add more routes when needed
 ];
+
+export const TAB_NEW_ARRIVALS = "newArrivals";
```

Comment:
Can be just `NEW_ARRIVALS`, but if you want to have TAB, I think it's more natural to be `NEW_ARRIVALS_TAB`